### PR TITLE
Fix nightly clippy warnings

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

File: cairo/src/font/font_options.rs [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 #[cfg(feature = "v1_16")]
 use std::ffi::CString;
+use std::hash;
 #[cfg(not(feature = "use_glib"))]
 use std::ptr;
-use std::{cmp::PartialEq, hash};
```
Comment (sdroege) [inline cairo/src/font/font_options.rs:7]: Why are you splitting this?

**Review by sdroege — COMMENTED** [review]
While you're at it, you could also remove the "unused import: `std::os::raw::c_int`" on Windows :)

Looks good to me otherwise

File: cairo/src/font/font_options.rs (hunk 1 above)
Comment (bilelmoussaoui) [reply cairo/src/font/font_options.rs:7]: i didn't split it, removed cmp::PartialEq as it is already in scope

Comment (sdroege) [reply cairo/src/font/font_options.rs:7]: Oh and that then reordered `std::hash`, I see. Sorry :)

**Comment by bilelmoussaoui** [comment]
> While you're at it, you could also remove the "unused import: `std::os::raw::c_int`" on Windows :)
> 
> Looks good to me otherwise

Done that along with another fix that shows on the CI.

File: gio/src/subclass/application.rs [hunk 2]
```diff
@@ -417,14 +417,16 @@ mod tests {
 
         impl ApplicationImpl for SimpleApplication {
             fn command_line(&self, cmd_line: &crate::ApplicationCommandLine) -> ExitCode {
```
[the remark this answers is not in the document]
Comment (sdroege) [reply gio/src/subclass/application.rs:419]: (only on Windows of course)

[the remark this answers is not in the document]
Comment (bilelmoussaoui) [reply gio/src/subclass/application.rs:420]: meh, switched the guard to the whole vfunc instead

[the remark this answers is not in the document]
Comment (sdroege) [reply gio/src/subclass/application.rs:420]: Yeah but that also defines the exit code so now it fails on Windows :upside_down_face: 

```
---- subclass::application::tests::test_simple_application stdout ----
thread 'subclass::application::tests::test_simple_application' panicked at gio\src\subclass\application.rs:468:9:
assertion `left == right` failed
  left: ExitCode(1)
 right: ExitCode(20)
```

[the remark this answers is not in the document]
Comment (bilelmoussaoui) [reply gio/src/subclass/application.rs]: annoying, fixed

**Review by sdroege — APPROVED** [review]

**Pull request merged**